web/src/components/core/IssuesDialog.jsx

```diff
@@ -144,15 +118,40 @@ const IssuesContent = ({ issues }) => {
 };
 
 /**
- * Page to show all issues.
+ * Popup to show more issues details from the installation overview page.
+ *
+ * It initially shows a loading state,
+ * then fetches and displays a list of issues of the selected category, either 'product' or 'storage' or 'software'.
+ *
+ * It uses a Popup component to display the issues, and an If component to toggle between
+ * a loading state and the content state.
+ *
  * @component
+ *
+ * @param {object} props
+ * @param {boolean} [props.isOpen] - A boolean value used to determine wether to show the popup or not.
+ * @param {function} props.onClose - A function to call when the close action is triggered.
+ * @param {string} props.sectionId - A string which indicites which type of issues is going to be shown in the popup.
  */
-export default function IssuesPage() {
+export default function IssuesDialog({ isOpen = false, onClose, sectionId }) {
   const [isLoading, setIsLoading] = useState(true);
-  const [issues, setIssues] = useState();
+  const [issues, setIssues] = useState([]);
   const client = useInstallerClient();
   const { cancellablePromise } = useCancellablePromise();
-  const [notification, updateNotification] = useNotification();
+  const titles = {
```

Comment:
Since we're going to drop the icon, as commented before, what about receiving the title as prop?